azkaban-common/src/main/java/azkaban/utils/StringUtils.java

```diff
@@ -88,4 +89,19 @@ public static String join2(Collection<String> list, String delimiter) {
 
     return buffer.toString();
   }
+
+  private static final Pattern BROWSWER_PATTERN = Pattern
+      .compile(".*Chrome.*|.*Firefox.*|.*Safari.*");
```

Comment:
Just wondering, are we going to try match these patterns case specific, or non case specific?